bin/_plugins/profile.js

```diff
@@ -114,11 +117,25 @@ function sandboxFromArguments(args, options) {
             return result;
         }
 
+        function onBeforeRequestRuntime(request) {
+            if (args.runtime) {
+                return request.set('x-wt-runtime', args.runtime);
+            }
+
+            return request;
+        }
+
         function onProfileLoaded(profile) {
             if (args.container) profile.container = args.container;
             if (args.url) profile.url = args.url;
             if (args.token) profile.token = args.token;
 
+            if (args.runtime) {
+                profile.onBeforeRequest = (profile.onBeforeRequest || []).concat(
```

Comment:
If `profile.onBeforeRequest` is defined here, is it guaranteed to always be an array? I couldn't figure that out

Reply:
It should based on https://github.com/auth0/sandboxjs/blob/master/lib/sandbox.js#L43 but I'll add a guard against just to be safer 👍